example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -21,89 +21,63 @@
 #include <cstdint>
 #include <iostream>
 
-//-----------------------------------------------------------------------------
-template<size_t width>
-ALPAKA_FN_ACC size_t linIdxToPitchedIdx(size_t const globalIdx, size_t const pitch)
-{
-    const size_t idx_x = globalIdx % width;
-    const size_t idx_y = globalIdx / width;
-    return idx_x + idx_y * pitch;
-}
-
 //#############################################################################
 //! Prints all elements of the buffer.
 struct PrintBufferKernel
 {
     //-----------------------------------------------------------------------------
-    template<typename TAcc, typename TData, typename TExtent>
+    template<typename TAcc, typename Data, typename Idx>
```

Comment:
alpaka is using for template parameter always `TName`

Reply:
I just ran clang-tidy to prefix the template parameters and I got over 50 corrections in other headers in alpaka that I did not touch. I tried to rename a few manually.